tests/testthat/test-pphcv.R

```diff
@@ -0,0 +1,187 @@
+context("Checking pre-processing of high-cardinality variables")
+
+set.seed(10)
+dat <- data.frame(Ycontinuous = sample(5:42, size = 14, replace = TRUE),
+                  Ycategorical = sample(c("Y", "N"), size = 14, 
+                                        replace = TRUE, prob = c(0.45, 0.55)),
+                  YCat01 = c(0, 0, 0, 0, 0, 1, 1, 1, 1, 0, 0, 1, 1, 0),
+                  hcX = sample(letters[1:5], size = 14, replace = TRUE),
+                  hcX2 = sample(letters[6:10], size = 14, replace = TRUE),
+                  hcX3 = sample(letters[11:15], size = 14, replace = TRUE),
+                  Ymulticlass = sample(c("blue", "green", "red"), size = 14, 
+                                       replace = TRUE))
+
+test_that("For a continuous target, one high-cardinality variable, and no 
+          specified m, pphcv returns correct, new feature", {
+
+  b <- pphcv(df = dat, target = "Ycontinuous", high_card = "hcX", 
+             type = "continuous")
+  
+  expect_equal(b$newdf$hcX_new[1], 22.05357, tolerance = 0.001)
+  expect_equal(b$newdf$hcX_new[12], 20.55357, tolerance = 0.001)
+})
```

Comment:
The indentation here above is good. Would you mind applying to all tests in this series?